models/sitemealtype.js

```diff
@@ -9,13 +9,12 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      SiteMealType.belongsTo(models.Site);
+      SiteMealType.belongsTo(models.MealType);
     }
   }
   SiteMealType.init(
-    {
-      SiteId: DataTypes.INTEGER,
-      MealTypeId: DataTypes.INTEGER,
-    },
+    {},
```

Comment:
Once a belongsTo relationship has been added, the id column can be removed from the attributes list, since it will be inferred from the relationship.